diff --git a/src/file-01.ts b/src/file-01.ts
new file mode 100644
index 0000000..1111111
--- /dev/null
+++ b/src/file-01.ts
@@ -0,0 +1,2 @@
+export const file01a = 1;
+export const file01b = 2;
diff --git a/src/file-02.ts b/src/file-02.ts
new file mode 100644
index 0000000..1111112
--- /dev/null
+++ b/src/file-02.ts
@@ -0,0 +1,2 @@
+export const file02a = 1;
+export const file02b = 2;
diff --git a/src/file-03.ts b/src/file-03.ts
new file mode 100644
index 0000000..1111113
--- /dev/null
+++ b/src/file-03.ts
@@ -0,0 +1,2 @@
+export const file03a = 1;
+export const file03b = 2;
diff --git a/src/file-04.ts b/src/file-04.ts
new file mode 100644
index 0000000..1111114
--- /dev/null
+++ b/src/file-04.ts
@@ -0,0 +1,2 @@
+export const file04a = 1;
+export const file04b = 2;
diff --git a/src/file-05.ts b/src/file-05.ts
new file mode 100644
index 0000000..1111115
--- /dev/null
+++ b/src/file-05.ts
@@ -0,0 +1,2 @@
+export const file05a = 1;
+export const file05b = 2;
diff --git a/src/file-06.ts b/src/file-06.ts
new file mode 100644
index 0000000..1111116
--- /dev/null
+++ b/src/file-06.ts
@@ -0,0 +1,2 @@
+export const file06a = 1;
+export const file06b = 2;
diff --git a/src/file-07.ts b/src/file-07.ts
new file mode 100644
index 0000000..1111117
--- /dev/null
+++ b/src/file-07.ts
@@ -0,0 +1,2 @@
+export const file07a = 1;
+export const file07b = 2;
diff --git a/src/file-08.ts b/src/file-08.ts
new file mode 100644
index 0000000..1111118
--- /dev/null
+++ b/src/file-08.ts
@@ -0,0 +1,2 @@
+export const file08a = 1;
+export const file08b = 2;
diff --git a/src/file-09.ts b/src/file-09.ts
new file mode 100644
index 0000000..1111119
--- /dev/null
+++ b/src/file-09.ts
@@ -0,0 +1,2 @@
+export const file09a = 1;
+export const file09b = 2;
diff --git a/src/file-10.ts b/src/file-10.ts
new file mode 100644
index 0000000..1111120
--- /dev/null
+++ b/src/file-10.ts
@@ -0,0 +1,2 @@
+export const file10a = 1;
+export const file10b = 2;
diff --git a/src/file-11.ts b/src/file-11.ts
new file mode 100644
index 0000000..1111121
--- /dev/null
+++ b/src/file-11.ts
@@ -0,0 +1,2 @@
+export const file11a = 1;
+export const file11b = 2;
diff --git a/src/file-12.ts b/src/file-12.ts
new file mode 100644
index 0000000..1111122
--- /dev/null
+++ b/src/file-12.ts
@@ -0,0 +1,2 @@
+export const file12a = 1;
+export const file12b = 2;
